src/scss/components/_media-element.scss

```diff
@@ -134,7 +134,7 @@
 
             .c-mediaElement--fullstack--negativeTop & {
                 flex: auto;
-                margin-top: -#{spacing(x2)};
+                margin-top: spacing(x4);
```

Comment:
The modifier part of the class name states `--negativeTop` and with this change it is not anymore. Should we rename this class name or add a new one with `spacing(x4)` top margin value instead?